frontend/src/pages/Test.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { Button, ButtonGroup, Container } from 'react-bootstrap';
+import { ENVIRONMENT } from 'constants/environment';
+import download from './../utils/download';
+import { useDispatch } from 'react-redux';
+
+/**
+ * Test provides a testing page for various things.
+ */
+const Test = () => {
```

Comment:
Test page that we can use for various things.  Temporarily using to show the download of the property report.